### ISSUE TITLE
How to create sorted list of Pages in Octopress or Jekyll?

### ISSUE THREAD

**Issue by tuxdna** [imathis/octopress#1517]
I have asked this question on StackOverflow: [here](https://stackoverflow.com/questions/22390572/how-to-create-sorted-list-of-pages-in-octopress-or-jekyll)

I have a Octopress website, which has posts and pages. Now I want to add another category of pages which I want to call as `writepus` ( notes which I will keep updating like a wiki  via git commits).

I want to keep these notes in a folder called `_notes`, just like we have `_posts` in `source` folder of Octopress.

I have a folder called `_writeups/` with files such as:

```
subject1.html
subject2.html
```

I have a file called `notes/list.html` with following content. 

```

---
layout: page
navbar: Notes
title: Notes
footer: false

---

<div id="blog-archives">
{% for post in site.writeups reverse %}
{% capture this_year %}{{ post.date | date: "%Y" }}{% endcapture %}
{% unless year == this_year %}
  {% assign year = this_year %}
  <h2>{{ year }}</h2>
{% endunless %}
<article class="page-header">
  {% include archive_post.html %}
</article>
{% endfor %}
</div>
```

Basically I want to create a listing of these writeups so that I can keep updating them as and when I get time. Also I want to keep these separate from `posts` and `pages`.

How can I achieve this functionality using Octopress / Jekyll ?
